spec/lib/local-links-manager/import/interactions_importer_spec.rb

```diff
@@ -51,5 +51,51 @@
         LocalLinksManager::Import::InteractionsImporter.new(csv_downloader).import_records
       end
     end
+
+    context 'when an interaction is no longer in the CSV' do
```

Comment:
Now that we have a separate spec for the `ImportComparer` class, I'm wondering if we can remove these tests here, since they test the same thing (expecting that the `Services` module receives the method `icinga_check`)? I'm not 100% sure though so happy to chat in person tomorrow.

If we do leave them, we should add some specs for the `LocalAuthority` importer too.